src/definitions.ts

```diff
@@ -761,6 +773,19 @@ export interface AuthDependencies {
   storage: IStorage<string>;
 }
 
+
+export interface IGoogleData {
+  webClientId: string;
+}
+
+export interface IFacebookNativeAuth {
+  authenticate(fields: string[]): Promise<any>;
+}
+
+export interface IGoogleNativeAuth {
+  authenticate(data?: any, options?: any): Promise<any>;
```

Comment:
`data` and `options` aren't used, can we just have the function take no parameters? Also we don't need to resolve with anything right now. You're thinking too much! 😄 We may want to change what's returned later or do something different altogether, which would be a _ton_ easier if this returned nothing for _right now_. I've found this strategy useful/interesting: https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it